### DISCUSSION TITLE
Mapzen alternative?

### DISCUSSION THREAD

**Discussion by aureateflux** [FacilMap/facilmap#250] [General]
This is a really interesting project, and I'm glad to see activity has resumed on it!

I decided to try to spin up a container with facilmap to help plan an upcoming trip, and I noticed that Mapzen is no longer available. 

Are there any alternatives to Mapzen or plans to implement another service to provide elevation information?

Thanks!

**Comment by cdauth** [comment]
I did a quick research for alternatives and stumbled upon Open Elevation. I have activated that now on https://beta.facilmap.org/ (any maps created/modified there will be deleted during the next release), search results and markers will now get elevation info.

**Reply by cdauth** [reply]
I have also implemented a migration that will retrieve the elevations for all existing markers.

This will be available with the release of FacilMap 4, which will hopefully happen in a few weeks.
